lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.h

```diff
@@ -257,6 +257,14 @@ class TypeSystemSwiftTypeRef : public TypeSystemSwift {
       bool print_help_if_available, bool print_extensions_if_available,
       lldb::DescriptionLevel level = lldb::eDescriptionLevelFull) override;
 
+  /// Recursively transform the demangle tree starting a \p node by
+  /// doing a post-order traversal and replacing each node with
+  /// fn(node).
+  static swift::Demangle::NodePointer Transform(
```

Comment:
can/should `Transform` be a static function and not exposed?

Reply:
No, you'll see why in the next patch :-)
(I need to call it in the expression evaluator)